README.md

```diff
@@ -1,2 +1,30 @@
 # Review-project
-Review your project from trials
+
+Review your project from trials ( Tribute Page )
+
+## Built With
+
+- Html
+- Css
+
```

Comment:
I kindly suggest you should include more details on how to download and install including the GitHub repo and instructions on how best to install and run. This will make it easy for references and enable others to easily understand your work